src/createStore.ts

```diff
@@ -107,12 +137,13 @@ export default function createStore<
     )
   }
 
-  let currentReducer = reducer
-  let currentState = preloadedState as S
-  let currentListeners: (() => void)[] | null = []
-  let nextListeners = currentListeners
-  let isDispatching = false
+  let currentReducer = reducer // set currentReducer to reducer set above
```

Comment:
NIT: Can this be changed to `defined` above instead of `set`? When I first read this I confused it with the `Set` object/keyword.

Reply:
Yep.